sda-commons-shared-wiremock-testing/src/test/java/org/sdase/commons/shared/wiremock/testing/WireMockExampleTest.java

```diff
@@ -0,0 +1,58 @@
+package org.sdase.commons.shared.wiremock.testing;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.getRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.okJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.github.tomakehurst.wiremock.junit5.WireMockExtension;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import jakarta.ws.rs.core.Response;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.sdase.commons.shared.wiremock.testing.testapp.WireMockConfig;
+import org.sdase.commons.shared.wiremock.testing.testapp.WireMockTestApp;
+
+class WireMockExampleTest {
+
+  @RegisterExtension
+  @Order(0)
+  static final WireMockExtension WIRE =
+      new WireMockExtension.Builder().options(wireMockConfig().dynamicPort()).build();
```

Comment:
`dynamicPort` is the default. Shouldn't we provide the simplest solution as example?

```suggestion
  static final WireMockExtension WIRE = new WireMockExtension.Builder().build();
```

Reply:
Removed dynamic port config in wiremock tests
https://github.com/SDA-SE/sda-dropwizard-commons/pull/2973